packages/components/src/card/style.scss

```
@@ -18,26 +18,21 @@
 .woocommerce-card__header {
 	padding: ($gap - 3) $gap;
 	border-bottom: 1px solid $core-grey-light-700;
-	display: grid;
+	display: flex;
 	align-items: center;
 
-	.has-action & {
-		grid-template-columns: auto 1fr;
+	>* {
```

I think you could use the class `.woocommerce-card__header-item`, so there is no need to use the universal selector here.
```suggestion
	> .woocommerce-card__header-item {
```

---

This seems like a safe commit to make since there doesn't seem to be any way of adding an item without `.woocommerce-card__header-item` so I'll go ahead and add this.

But curious if there are any reasons not to use the universal selector here since we would probably want any items directly under the header to share these styles.

---

Well, the universal selector is usually considered not to be very efficient:
https://css-tricks.com/efficiently-rendering-css/#article-header-id-1

In addition, in the future we might want to add a child to `.woocommerce-card__header` that is absolutely positioned or has [`display: contents`](https://caniuse.com/#search=display%3A%20contents). In those cases, if we use a class that must be manually specified instead of the universal selector, we will not need to reset the styles.